includes/ziti/ziti_log.h

```diff
@@ -43,6 +43,8 @@ enum DebugLevel {
 if (level <= ziti_log_level()) { ziti_logger(level, __FILE__, __LINE__, __func__, fmt, ##__VA_ARGS__); }\
 } while(0)
 
+#define ZTX_LOG(lvl, fmt, ...) ZITI_LOG(lvl, "ztx[%d] " fmt, ztx->id, ##__VA_ARGS__)
```

Comment:
do we really want this exposed to consumers?

Reply:
Ahh hmm. I don't know what would happen if they tried to use it. 

I can move it to `zt_internal.h`. Is that a better place for it @ekoby ?

Reply:
yes